apis/v1alpha2/httproute_types.go

```diff
@@ -408,6 +408,26 @@ type HTTPQueryParamMatch struct {
 	Value string `json:"value"`
 }
 
+// HTTPMethod describes how to select a HTTP route by matching the HTTP
+// method as defined by
+// [RFC 7231](https://datatracker.ietf.org/doc/html/rfc7231#section-4) and
+// [RFC 5789](https://datatracker.ietf.org/doc/html/rfc5789#section-2).
+// The value is expected in upper case.
+// +kubebuilder:validation:Enum=GET;HEADER;POST;PUT;DELETE;CONNECT;OPTIONS;TRACE;PATCH
```

Comment:
```suggestion
// +kubebuilder:validation:Enum=GET;HEAD;POST;PUT;DELETE;CONNECT;OPTIONS;TRACE;PATCH
```

This is already facing copy-paste challenges. 😞